src/redux/books/bookSlice.js

```diff
@@ -1,23 +1,107 @@
-import { createSlice } from '@reduxjs/toolkit';
-import BookItems from './BookItems';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+// import BookItems from './BookItems';
```

Comment:
- [ ] Please delete all commented-out code, it will make your codebase look unorganised and harder to read.